### DISCUSSION TITLE
Search Entries in the Packages

### DISCUSSION THREAD

**Discussion by engahmadkadi** [ENDER-9/Vocascan#104] [Ideas]
Hi Vocascan,

I wish to have the possibility to search the entries on the page of showing the list of packages, and inside the package.
This would ease the continuous enhancement of own vocabularies, for example to see if the entry is already entered, or to follow a previous example.

Thanks a lot!

**Comment by luwol03** [comment]
Hey engahmadkadi,

great idea to search for vocabularies. But on top of that there is already planned to show the user a little warning if he is trying to create similar vocabularies he already has in their package.

Can you please fire also an issue for that?

luwol03
